### PR TITLE
Added HTML link for each job name on targets page

### PR DESCRIPTION
**Pull request by drewhemm**
Useful for quick navigation on the target page when there are many jobs
and targets

---
This change is [Reviewable](https://reviewable.io/reviews/prometheus/prometheus/1856)

### CODE REVIEW

File: web/ui/templates/targets.html [hunk 1]
```diff
@@ -6,7 +6,7 @@ <h2 id="targets">Targets</h2>
       <table class="table table-condensed table-bordered table-striped table-hover">
         {{range $job, $pool := .Pools}}
           <thead>
-            <tr><th colspan="5" class="job_header">{{$job}}</th></tr>
+            <tr><th colspan="5" class="job_header"><a href="#{{$job}}">{{$job}}</a></th></tr>
```
Comment (brian-brazil) [inline web/ui/templates/targets.html]: You need a name or id to go with that. As-is this does nothing.

**Comment by drewhemm** [comment]
Whoops, been a while since I've done any HTML :)

File: web/ui/templates/targets.html (hunk 1 above)
Comment (drewhemm) [inline web/ui/templates/targets.html]: Whoops, been a while since I've done any HTML :)

**Comment by fabxc** [comment]
Thanks, can you run `make assets format` and squash your changes into a single commit?

**Comment by drewhemm** [comment]
As requested :)

**Comment by fabxc** [comment]
My bad, dropped the ball on this but don't want to bother you with rebasing.
Moved your commit to #1874 and closing here.

**Pull request closed without merging**